themes-default/slim/views/addShows_addExistingShow.mako

```diff
@@ -91,22 +70,37 @@ const startVue = () => {
             }
         },
         methods: {
+            rootDirsUpdated(value, data) {
+                this.rootDirs = data.map(rd => {
+                    return {
+                        selected: true,
+                        path: rd.path
+                    };
+                });
+            },
             async update() {
                 if (this.isLoading) return;
 
                 this.isLoading = true;
-                if (this.selectedRootDirs.length === 0) {
+
+                const indices = this.rootDirs
+                    .reduce((indices, rd, idx) => {
+                        if (rd.selected) indices.push(idx);
+                        return indices;
+                    }, []);
+                if (indices.length === 0) {
                     this.dirList = [];
                     this.isLoading = false;
                     return;
                 }
-                const params = { 'root-dir': JSON.stringify(this.selectedRootDirs) };
+
+                const params = { 'root-dirs': indices.join(',') };
```

Comment:
query parameters should be camelCase

Reply:
I think I searched the code and there wasn't any existing endpoint that had a multi-word parameter name. I'll change it.